man/sg.1.xml

```diff
@@ -70,6 +75,38 @@
     </para>
   </refsect1>
 
+  <refsect1 id='options'>
+    <title>OPTIONS</title>
+    <para>
+      The options which apply to the <command>sg</command> command are:
+    </para>
+    <variablelist remap='IP'>
+      <varlistentry>
+	<term><option>-</option>, <option>-l</option></term>
+	<listitem>
+	  <para>
+	    Provide an environment similar to what the user would expect had
+	    the user logged in directly.
+	  </para>
+	</listitem>
+      </varlistentry>
+      <varlistentry>
+	<term>
+	  <option>-c</option>
+	</term>
+	<listitem>
+	  <para>
+	    Specify a command that will be invoked by the shell using its
+	    <option>-c</option>.
+	  </para>
+	  <para>
+	    This is the default. For backward compatibility.
```

Comment:
This should use `;` or `,` instead of `.`

Reply:
The current way fits `CHFN_RESTRICT.xml`. Adjusted anyway